shared/fs/banner/fileui-banner/index.js

```diff
@@ -27,6 +29,11 @@ const Banner = ({kbfsEnabled, showBanner, onInstall, onDismiss, openInFileUI}: P
     alignItems: 'center',
     position: 'relative',
   }
+  const promptText = kbfsOutdated ? ( dokanUninstall ?
```

Comment:
Not sure you need the parenthesis. Usually we format these like:
```javascript
const promptText = kbfsOutdated
  ? dokanUninstall
    ? 'A newer version...'
    : 'A newer version...'
  : 'Get access to your files...'
```

Reply:
missed this one. ok.